okhttp/src/main/java/okhttp3/internal/platform/AndroidPlatform.java

```diff
@@ -40,6 +42,11 @@
 class AndroidPlatform extends Platform {
   private static final int MAX_LOG_LENGTH = 4000;
 
+  private static final OptionalMethod<Socket> getAlpnMethod
```

Comment:
Is there a real need to hold these two guys as static final? 

Platform is usually built once, look like you can move this into `buildIfSupported()` as local variables.